.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -42,3 +42,10 @@ Enter your extended release note in the block below. If the PR requires addition
 ```release-note
 
 ```
+<!--
+**Additional documentation, like links to enhancement proposals (KEP)**:
+Please use the following format for linking documentation: `- [DESCRIPTION] [LINK]`
+```docs
+- Relevant KEP https://github.com/kubernetes/enhancements/...
```

Comment:
Haven't actually seen the code diff in k/release yet, but would it be useful to have a delimiter between `[DESCRIPTION]` and `[LINK]` to come to know when the description ends and when the link begins? Kinda rare, but I'm wondering what would happen if a description has a link... :thinking: 

Also, can we have multiple links for one description? I'm asking this from a completely noob perspective :) .. for example, I wouldn't know if I'm allowed to do a `Related doc PRs <link> <link> <link>`.

Reply:
I created a follow up PR for stripping away separators like `-` and `:` here: https://github.com/kubernetes/release/pull/748 This should solve the delimiter problem :)

Hm, multiple links to a single entry are currently not supported, for that we would have to create a new line, which equals a new entry. What works is omiting the description, like:
```docs
- https://google.com
```